Meta/Utils/vcpkg_deps.py

```
@@ -0,0 +1,539 @@
+            fifo_fd = os.open(token_fifo, os.O_RDWR)
+            os.write(fifo_fd, b"x" * tokens)
+
+            ninja_environment = os.environ.copy()
+            ninja_environment["LADYBIRD_VCPKG_TOKEN_FIFO"] = str(token_fifo)
+            ninja_environment["LADYBIRD_VCPKG_TOKEN_POOL"] = str(tokens)
+
+            try:
+                ninja_arguments = ["ninja", "-C", str(deps_dir)]
+                if jobs:
+                    ninja_arguments.extend(["-j", jobs])
+                subprocess.check_call(ninja_arguments, env=ninja_environment)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)